cleanlab/internal/validation.py

```diff
@@ -18,12 +18,18 @@
 Checks to ensure valid inputs for various methods.
 """
 
+from typing import Any, List, Optional, Union
 import warnings
 import numpy as np
 import pandas as pd
 
 
-def assert_valid_inputs(X, y, pred_probs=None, multi_label=False):
+def assert_valid_inputs(
+    X: Any,
```

Comment:
I understand why we're using `Any` here, because we don't want to spell out all the different possibilities / we want to allow certain duck-typed things without having Cleanlab depend on extra libraries, but "Any" doesn't read great in type annotations. What if we create a type alias for this, perhaps in a new `cleanlab.typing` module, and call it `dataset_like` (which will just be Any):

```python
from typing import TypeAlias

dataset_like: TypeAlias = Any
```

Reply:
No, I don't think we need type aliases for those. Probabilities are floats (that are between 0 and 1), scores are floats (that are between 0 and 1), a joint is a 2d numpy array of floats.